packages/element/package.json

```diff
@@ -26,8 +26,8 @@
 		"@babel/runtime": "^7.9.2",
 		"@wordpress/escape-html": "file:../escape-html",
 		"lodash": "^4.17.15",
-		"react": "^16.9.0",
-		"react-dom": "^16.9.0"
+		"react": "16.11.0",
```

Comment:
We should allow ranges here.

If Travis will continue to fail PHP code needs to be updated as well. At the moment it will use the version that WordPress core uses. See related changes from the past:
https://github.com/WordPress/gutenberg/pull/16982

Otherwise, it can be tackled separately.

Reply:
I have a PR to upgrade React here https://github.com/WordPress/gutenberg/pull/21289 
It's blocked though because we have a compatibility issue with the data module

Reply:
RN might not work with 16.13.x, you would have to test it. However this PR as is might be fine to land as mentioned earlier.

Reply:
Thanks @gziolo, I have returned the range.